libraries/Matter/src/MatterEndpoints/MatterDimmableLight.cpp

```diff
@@ -0,0 +1,158 @@
+#include <sdkconfig.h>
```

Comment:
Please add the Espressif's license header (IDK if the other Matter files might be missing it)

Reply:
I'll do that latter for all the files.

Reply:
done to all source code files.